pkg/providers/aws/cluster_network_provider.go

```diff
@@ -1157,6 +1176,30 @@ func (n *NetworkProvider) reconcileRDSVpcConfiguration(ctx context.Context, priv
 	return nil
 }
 
+//this function removes tags and reads them to an rds subnet group
+func (n *NetworkProvider) updateSubnetGroupTags(foundSubnet *rds.DBSubnetGroup, croOwnerTag *rds.Tag, logger *logrus.Entry) error {
+	logger.Info()
+	_, err := n.RdsApi.RemoveTagsFromResource(&rds.RemoveTagsFromResourceInput{
+		ResourceName: aws.String(*foundSubnet.DBSubnetGroupArn),
+		TagKeys: []*string{
+			aws.String(*croOwnerTag.Key),
+		},
+	})
+	if err != nil {
+		return errorUtil.Wrap(err, "error updating db subnet group tags")
+	}
+	_, err = n.RdsApi.AddTagsToResource(&rds.AddTagsToResourceInput{
+		ResourceName: aws.String(*foundSubnet.DBSubnetGroupArn),
```

Comment:
Same here with the dereferencing, we can do:

```suggestion
		ResourceName: foundSubnet.DBSubnetGroupArn,
```